src/test/java/tests/WishlistTest.java

```diff
@@ -0,0 +1,61 @@
+package tests;
+
+import io.qameta.allure.Description;
+import org.junit.jupiter.api.*;
+import pages.*;
+
+public class WishlistTest extends BaseTest {
+
+    private LoginPage loginPage;
+    private WishlistPage wishlistPage;
+    private ProductPage productPage;
+    private AccountPage accountPage;
+    private String wishlist;
+    private String product = "Blouse";
+
+    @BeforeEach
+    public void login() {
+        loginPage = new LandingPage(driver).clickSignInButton();
+        loginPage.login(user.getEmail(), user.getPassword());
+        wishlistPage = new WishlistPage(driver);
+        productPage = new ProductPage(driver);
+        accountPage = new AccountPage(driver);
+    }
+
+    @Description("Wishlist was created automatically")
+    @Test
+    public void wishlistCanBeCreatedAutomatically() {
+        wishlist = "My wishlist";
+        accountPage.clickProductsButton();
```

Comment:
pt2 for comment above:
productPage = accountPage.clickProductsButton();
        productPage.addProductToWishlist(product).clickAccountNameButton();
        wishlistPage = accountPage.clickWishlistButton();

Reply:
applies to tests below as well